src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/TagHelperServiceTestBase.cs

```diff
@@ -203,9 +203,9 @@ public TagHelperServiceTestBase()
 
         protected ILogger Logger { get; } = NullLogger.Instance;
 
-        internal static RazorCodeDocument CreateCodeDocument(string text, params TagHelperDescriptor[] tagHelpers)
+        internal static RazorCodeDocument CreateCodeDocument(string text, bool isRazorFile, params TagHelperDescriptor[] tagHelpers)
         {
-            return CreateCodeDocument(text, CSHtmlFile, tagHelpers);
+            return CreateCodeDocument(text, isRazorFile ? RazorFile : CSHtmlFile, tagHelpers);
```

Comment:
Curious if this was papering over any bugs?

Reply:
This specific call path in semantic tokens is only used to get the C# tokens, so surprisingly there weren't any bugs